webpack.config.js

```diff
@@ -28,6 +28,13 @@ module.exports = {
         enforce: "pre",
         test: /\.js$/,
         loader: "source-map-loader"
+      },
+      {
+        test: /\.(png|jpe?g|gif|jp2|webp)$/,
```

Comment:
This knows to use the file loader to load the images if they're in that location.